### PR TITLE
Player Death Scene

### PR DESCRIPTION
**Pull request by fjz345**
There is a test that kills the player soon after start.

### CODE REVIEW

**Review by AntonAsbrink — CHANGES_REQUESTED** [review]
Crashes when closing the window using ESC

**Review by AntonAsbrink — DISMISSED** [review]

File: StortSpelProjekt/Engine/src/ECS/SceneManager.h [hunk 1]
```diff
@@ -36,7 +42,12 @@ class SceneManager
 	std::vector<Scene*> m_ActiveScenes;
 	std::set<Scene*> m_LoadedScenes;
 
+	Scene* m_GameOverScene = nullptr;
```
Comment (fredrix96) [inline StortSpelProjekt/Engine/src/ECS/SceneManager.h]: You forgot a p for pointer

**Review by fredrix96 — CHANGES_REQUESTED** [review]
Just a small code standard error, the rest looks to work well

**Review by fredrix96 — APPROVED** [review]
Seems to work as intended. Well done!

**Review by AntonAsbrink — APPROVED** [review]

**Pull request merged**